src/OpenSpaceNet.cpp

```diff
@@ -325,7 +346,7 @@ void OpenSpaceNet::initFilter()
             } else if (action == "exclude") {
                 if (firstAction) {
                     OSN_LOG(info) << "User excluded regions first...automatically including the bounding box...";
-                    regionFilter_->add({{bbox_}});
+                    regionFilter_->add(Polygon(LinearRing(cv::Rect(0, 0, bbox_.width, bbox_.height))));
```

Comment:
No, that will break Mac build. Also this doesn't work. It's a GCC bug that this was allowed at all.